packages/main/src/dynamic-date-range-options/LastNextTemplate.tsx

```diff
@@ -0,0 +1,84 @@
+import type DynamicDateRange from "../DynamicDateRange.js";
+import type { DynamicDateRangeValue } from "../DynamicDateRange.js";
+import StepInput from "../StepInput.js";
+import Select from "../Select.js";
+import type { SelectChangeEventDetail } from "../Select.js";
+import Option from "../Option.js";
+import Label from "../Label.js";
+import type LastOptions from "./LastOptions.js";
+import type NextOptions from "./NextOptions.js";
+
+export default function LastNextTemplate(this: DynamicDateRange) {
+	const currentOption = this._currentOption;
+	if (!currentOption) {
+		return <div>No option selected</div>;
+	}
+
+	// Check if there are multiple options available (grouped)
+	const availableOptions = (currentOption as LastOptions | NextOptions).availableOptions;
+	const isGrouped = Boolean(availableOptions && availableOptions.length > 1);
```

Comment:
!!?

Reply:
Probably an AI suggestion, that I have missed to 'polish' while polishing the code. Will fix it :D